### DISCUSSION TITLE
Find seller partner is using MFN or FBA for shipping?

### DISCUSSION THREAD

**Discussion by Nilanth** [amzn/selling-partner-api-models#3143] [General]
Is there any API to find the shipping method of the seller partner?  MFN or FBA or Both.

**Comment by NetRushJacob** [comment]
I don't think this would be a specific setting on the selling partner; they could be using FBA for a set of products and MFN for a set of products; they could even be toggling between the two. 

Orders API will tell you the fulfillment channel that orders are being fulfilled through.
Listing information might contain indicators of where the product is being fulfilled through.
If there's an API to list out / manage shipping templates, that might indicate something along these lines, but not exactly.
